File: models/src/main/java/ar/edu/itba/paw/models/constants/PetStatus.java
package ar.edu.itba.paw.models.constants;

public enum PetStatus {
    AVAILABLE(0),
    REMOVED(1),
    SOLD(2),
    UNAVAILABLE(3);

    final private int value;

    PetStatus(int value) {
        this.value = value;
    }

    public int getValue() {
        return this.value;
    }
}

File: webapp/src/main/reactjs/src/constants/petStatus.js
export const petStatus = {
    AVAILABLE: 0,
    REMOVED: 1,
    SOLD: 2,
    UNAVAILABLE: 3
};

export const petStatusToString = status => {
    switch (status) {
        case petStatus.AVAILABLE:
            return 'available';
        case petStatus.REMOVED:
            return 'removed';
        case petStatus.SOLD:
            return 'sold';
        case petStatus.UNAVAILABLE:
            return 'unavailable';
    }
};
